doc/calcurse.1.txt

```diff
@@ -30,519 +30,714 @@
-  See the 'FORMAT STRINGS' section for detailed information on format strings.
-
-Format strings
---------------
-
-Format strings are composed of printf()-style format specifiers -- ordinary
-characters are copied to stdout without modification. Each specifier is
-introduced by a *%* and is followed by a character which specifies the field to
-print. The set of available fields depends on the item type.
-
-Format specifiers for appointments
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-*s*::
-  Print the start time of the appointment as UNIX time stamp
-*S*::
-  Print the start time of the appointment using the *hh:mm* format
-*d*::
+[horizontal]
+*%d*::
   Print the duration of the appointment in seconds
-*e*::
-  Print the end time of the appointment as UNIX time stamp
-*E*::
-  Print the end time of the appointment using the *hh:mm* format
-*m*::
+*%e*::
+  Print the end time of the appointment as the Unix time in seconds
+*%E*::
+  Print the end time of the appointment or the marker +..:..+ if
+  it ends after midnight
+*%m*::
   Print the description of the item
-*n*::
+*%n*::
   Print the name of the note file belonging to the item
-*N*::
+*%N*::
   Print the note belonging to the item
-
-Format specifiers for events
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-*m*::
+*%r*::
+  Print the remaining time before the appointment
+*%s*::
+  Print the start time of the appointment as the Unix time in seconds
+*%S*::
+  Print the start time of the appointment or the marker +..:..+ if it begins
+  before midnight
+
+.+events+
+
+[horizontal]
+*%m*::
   Print the description of the item
-*n*::
+*%n*::
   Print the name of the note file belonging to the item
-*N*::
+*%N*::
   Print the note belonging to the item
 
-Format specifiers for todo items
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.+todo items+
 
-*p*::
-  Print the priority of the item
-*m*::
+[horizontal]
+*%m*::
   Print the description of the item
-*n*::
+*%n*::
   Print the name of the note file belonging to the item
-*N*::
+*%N*::
   Print the note belonging to the item
+*%p*::
+  Print the priority of the item
+
+[[_default_format_strings]]
+.Default format strings
+
+For each format option there is a default format string which is used when the option
+is not given. In query results the default format options are:
+
+  --format-apt " - %S -> %E\n\t%m\n"
+  --format-recur-apt " - %S -> %E\n\t%m\n"
+  --format-event " * %m\n"
+  --format-recur-event " * %m\n"
+  --format-todo "%p. %m\n"
+
+In all other cases (grep and dump-imported) the default format string is +"%(raw)"+.
+
+[[_extended_format_specifiers]]
+.Extended format specifiers
+
+Extended format specifiers can be used to control the printing of times for
+some of the single-letter specifiers. Additionally there are two specifiers
+that do not have any corresponding short form and are intended for use in
+scripting.
+
+*%d* ~ *%(duration*`[`*:*'format'`]`*)*::
+*%r* ~ *%(remaining*`[`*:*'format'`]`*)*::
```

Comment:
I see you intentation here, but even as a calcurse developer I was confused by this syntax. Can we simply drop the short versions here and add something like "%(duration) is the long version of %d and %(remaining) is the long version of %r" instead?